### ISSUE TITLE
amp-analytics implementation issues on amproject.org

### ISSUE THREAD

**Issue by dspattison-google** [ampproject/docs#25]
The hits being sent to google-analytics.com are missing the &cid parameter.  This hits are being dropped and not tracked.

example page:
https://www.ampproject.org/docs/reference/extended/amp-analytics.html
using the debug server for https://www.google-analytics.com/debug/collect?v=1&_v=a0&aip=true&_s=&dl=https%3A%2F%2Fwww.ampproject.org%2Fdocs%2Freference%2Fextended%2Famp-analytics.html&dt=amp-analytics&sr=2560x1440&_utmht=14540832363216&jid=&cid=&tid=UA-67833617-1&t=pageview&_r=1&z=0.0456467
```
{
  "hitParsingResult": [ {
    "valid": false,
    "parserMessage": [ {
      "messageType": "INFO",
      "description": "IP Address from this hit was anonymized to 72.241.118.0.",
      "messageCode": "VALUE_MODIFIED"
    }, {
      "messageType": "INFO",
      "description": "A default value was used for the parameter 'cid'. Please see http://goo.gl/a8d4RP#cid for details.",
      "messageCode": "DEFAULT_VALUE_USED",
      "parameter": "cid"
    }, {
      "messageType": "ERROR",
      "description": "A value is required for parameter 'cid'. Please see http://goo.gl/a8d4RP#cid for details.",
      "messageCode": "VALUE_REQUIRED",
      "parameter": "cid"
    } ],
    "hit": "/debug/collect?v=1\u0026_v=a0\u0026aip=true\u0026_s=\u0026dl=https%3A%2F%2Fwww.ampproject.org%2Fdocs%2Freference%2Fextended%2Famp-analytics.html\u0026dt=amp-analytics\u0026sr=2560x1440\u0026_utmht=1454083780209\u0026jid=\u0026cid=\u0026tid=UA-67833617-1\u0026t=pageview\u0026_r=1\u0026z=0.07820693154899105"
  } ],
  "parserMessage": [ {
    "messageType": "INFO",
    "description": "Found 1 hit in the request."
  } ]
}
```

**Comment by avimehta** [comment]
The fix should be live with next release.

**Comment by avimehta** [comment]
this is fixed now..

**Comment by rudygalfi** [comment]
Fixed via #1627.

**Comment by Meggin** [comment]
Closing as this works.
